src/components/AppDetails/ManageInstalledVersion.js

```diff
@@ -74,10 +80,22 @@ export const ManageInstalledVersion = ({
         <div className={styles.manageInstalledVersion}>
             {canInstall && (
                 <>
-                    <Button primary onClick={handleInstall}>
-                        {canUpdate
-                            ? i18n.t('Update to latest version')
-                            : i18n.t('Install')}
+                    <Button
+                        primary
+                        onClick={handleInstall}
+                        disabled={isInstalling}
+                    >
+                        {isInstalling && (
+                            <>
+                                {i18n.t('Installing...')}
+                                <CircularLoader small />
+                            </>
+                        )}
+                        {!isInstalling
+                            ? canUpdate
+                                ? i18n.t('Update to latest version')
+                                : i18n.t('Install')
+                            : ''}
```

Comment:
The button has an `icon` property, then it'd be in line with all other icon-buttons.
Also applies to buttons in files below